app/models/course/assessment/category.rb

```diff
@@ -40,7 +42,7 @@ def build_initial_tab
 
   def build_initial_folder
     build_folder(name: title, course: course, parent: course.root_folder,
-                 start_at: Time.zone.now)
+                 start_at: Time.zone.now) unless folder
```

Comment:
Oops, yep, I can take this out now, thanks !

I didn't split them before was because that `start_at` was set only when category was first created.